metrics.py

```diff
@@ -58,6 +58,7 @@ def get(self):
 
       memcache.set(KEY, data, time=CACHE_AGE)
 
+    data = self._clean_data(data)
```

Comment:
That's what I wanted to do initially but then I realized JSONHandler was also used in server.py to serve features.json. See https://github.com/GoogleChrome/chromium-dashboard/blob/3f8b0dc514cc86a532a8789da2539b4d117c373c/server.py#L112